features/api-layer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -24,6 +24,7 @@
   <reference id="eventConfDao" interface="org.opennms.netmgt.config.api.EventConfDao" availability="mandatory"/>
   <reference id="eventForwarder" interface="org.opennms.netmgt.events.api.EventForwarder" availability="mandatory"/>
   <reference id="eventSubscriptionService" interface="org.opennms.netmgt.events.api.EventSubscriptionService"/>
+  <reference id="onmsTopologyDao" interface="org.opennms.netmgt.topologies.service.api.OnmsTopologyDao" availability="mandatory"/>
```

Comment:
Is this DAO available if the Enlinkd service is disabled in `service-configuration.xml`? If not, we'll need make it optional, otherwise the whole `api-layer` feature won't start when the service is disabled.

Reply:
(We also have to fix that problem with some of the existing service references.)

Reply:
Will have to check on that.

Reply:
With the EnhancedLinkd service disabled the feature still starts up and the EdgeDao service is made available.